src/storage/lock_manager/deadlock.rs

```diff
@@ -396,12 +409,36 @@ impl Detector {
         })
         .map_err(|e| panic!("unexpected err: {:?}", e));
         handle.spawn(timer);
-        self.is_membership_change_monitor_scheduled = true;
+    }
+
+    fn schedule_detect_table_expiration(&self, handle: &Handle) {
+        info!("schedule detect table expiration");
+        let inner = Rc::clone(&self.inner);
+        let timer = Interval::new_interval(Duration::from_millis(TXN_DETECT_INFO_TTL))
```

Comment:
Could we use `on_tick` instead of a new interval in this situation?

Reply:
It's a future worker, not worker...